src/octoprint/printer/standard.py

```diff
@@ -369,10 +369,14 @@ def extrude(self, amount, speed=None, *args, **kwargs):
 
 		# Use specified speed (if any)
 		extrusion_speed = speed
```

Comment:
This assignment becomes useless after your changes

Reply:
Fixed. Thanks